### DISCUSSION TITLE
BETA 110 - Why implement TreeView ?????

### DISCUSSION THREAD

**Discussion by camomille22003** [twinbasic/twinbasic#1147] [General]
BETA 110 - Why implement TreeView which is not a basic component of VB6 (included in the Microsoft Windows Common Controls 6.0 (SP6) component), while still missing the main ones #938

**Comment by WaynePhillipsEA** [comment]
Don't worry, all controls are being worked on this month.  Not much longer to wait now.

Sometimes things get prioritised based on particular customer needs, or sometimes even for my own needs.  TreeView is an important control, and we have no free 64 bit version available, until now.

**Comment by mansellan** [comment]
Treeview is an important component, so is ListView. Just because they weren't shipped in-box with VB6 doesn't mean they're unloved. The first thing that annoyed me about VBA back in 1999 was a lack of the Treeview control. I fondly remember trying to explain to corporate IT why a mere admin drone like me needed a VS6 license.

Hoping we can put together a list of native controls on the wiki. Intrinsic and typical, MsComCtl and MsComCt2, likely others... broken down into controls with %member coverage,

Oh damn, did I just volunteer?

**Reply by WaynePhillipsEA** [reply]
Here's a list of all of the controls that shipped with VB6:   https://jeffpar.github.io/kbarchive/kb/194/Q194784/
